yapapi/ctx.py

```diff
@@ -129,13 +129,15 @@ def _payment_model(self) -> ComLinear:
 
         return self.__payment_model
 
-    def new_script(self):
+    def new_script(
```

Comment:
I'm not sure if this is a necessary change, we can just:
```
script = ctx.new_script()
script.timeout = timedelta(7)
```

(In general, I think it is better to have less functions with copied signatures)

Either way, the `params` section in the docstring is missing.

Reply:
I well, I prefer to pass params rather then tweak the resultant object ...

well, originally, I put `*args, **kwargs` here _exactly_ so as not to copy the signature but @zakaprov requested the params to be present on the `new_script` function

Reply:
I don't really understand you preference (if object has modifiable attribute, than changing it is a part of its API, so why not use it?). But well, OK.